include/config/overworld.h

```diff
@@ -3,6 +3,7 @@
 
 // Movement config
 #define OW_RUNNING_INDOORS          GEN_LATEST  // In Gen4+, players are allowed to run indoors.
+#define SLOW_MOVEMENT_ON_STAIRS     TRUE        // If enabled, the player will move slower up/down stairs like in FR
```

Comment:
Since it's limited to FRLG (at least in Gen3), I would personally vouch for having this off by default